server/src/main/java/org/opensearch/cluster/action/index/MappingUpdatedAction.java

```diff
@@ -157,17 +157,14 @@ protected void sendUpdateMapping(Index index, Mapping mappingUpdate, ActionListe
         putMappingRequest.source(mappingUpdate.toString(), XContentType.JSON);
         putMappingRequest.clusterManagerNodeTimeout(dynamicMappingUpdateTimeout);
         putMappingRequest.timeout(TimeValue.ZERO);
-        if (clusterService.state().nodes().getMinNodeVersion().onOrAfter(LegacyESVersion.V_7_9_0)) {
+        if (clusterService.state().nodes().getMinNodeVersion().onOrAfter(Version.V_3_0_0)) {
```

Comment:
This flipped logic for >= 3.0, preserving <= 7.9.0 behavior for >= 3.0. Was this intentional?

Reply:
I removed the version check altogether since it's no longer executed.